deployments/charts/kuma/templates/pre-upgrade-install-missing-crds-job.yaml

```diff
@@ -0,0 +1,93 @@
+{{- if .Values.installNewCrdsWhenUpgrade }}
+  {{- $serviceAccountName := printf "%s-install-missing-crds" (include "kuma.name" .) }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ $serviceAccountName }}
+  annotations:
+    "helm.sh/hook": "pre-upgrade"
+    "helm.sh/hook-weight": "-1"
+    "helm.sh/hook-delete-policy": "before-hook-creation,hook-succeeded,hook-failed"
+  labels:
+  {{- include "kuma.labels" . | nindent 4 }}
+{{- with .Values.hooks }}
+{{- with .preUpgrade }}
+{{- with .installMissingCrds }}
+{{- with .serviceAccount }}
+{{- with .imagePullSecrets }}
```

Comment:
Can we reduce this nesting? Maybe just `.Values.installMissingCrds.imagePullSecrets`? If we set a default value in the `values.yaml` file for the key, we could also remove the nil checks.  What do you think about the below structure?

```yaml
installCrdsOnUpgrade:
  enabled: true
  onlyMissing: true
  imagePullSecrets: []
```

Reply:
Addressed :-)